tests/metrics/eval_clients/_openai.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import annotations
+
+import json
+import os
+from unittest.mock import Mock, patch
+
+from openai.types.chat import ChatCompletion
+
+from langcheck.metrics.eval_clients import (AzureOpenAIEvalClient,
+                                            OpenAIEvalClient)
+
+
+def test_get_text_response_openai():
+    prompts = ['What is the capital of France?'] * 2
+    answer = 'Paris is the capital of France.'
```

Comment:
```suggestion
    prompts = ['Assess the factual consistency of the generated output...'] * 2
    answer = 'The output is fully factually consistent.'
```
nit: just to make the example a bit more consistent with what the prompts and answer will actually be like